src/Hazelcast.Net/DistributedObjects/DistributedObjectBase.cs

```diff
@@ -128,7 +128,7 @@ public Action<DistributedObjectBase> ObjectDisposed
         /// <summary>
         /// Gets the logger factory.
         /// </summary>
-        protected ILoggerFactory LoggerFactory { get; }
+        internal ILoggerFactory LoggerFactory { get; }
```

Comment:
Making this `internal` visible, along with the `SqlService` in `HMap`, feels like it's breaking encapsulation... I am not too sure... since it's all internal we can merge it but I'd like we think about it.

Reply:
we will think about it later